api/src/modules/Session/model.ts

```diff
@@ -36,6 +41,40 @@ export const SessionCreateOutput = objectType({
   },
 });
 
+// Actions
```

Comment:
Feel free to put it into the `Session/helpers.ts` file, to not expend this file over the limit